alpha/apps/kaltura/lib/batch2/bcdl/kBusinessConvertDL.php

```diff
@@ -22,6 +22,19 @@ public static function replaceEntry(entry $replacedEntry, entry $replacingEntry
 			return;
 		}
 
+		$lock = kLock::create("replacement_" . $replacedEntry->getId() . "_" . $replacingEntry->getId());
+		if ($lock && !$lock->lock())
+		{
+			KalturaLog::debug("Could not add a lock");
```

Comment:
The message is confusing, rephrase - could not lock + print the lock key as informative info